.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +0,0 @@
-### What
```

Comment:
I don't think this should be deleted. It will be used when a PR is created in  Github?

Reply:
This was by accident. Sorry! But the file has been restored Done in PR https://github.com/ONSdigital/dp-recipe-api/pull/127